kakao/src/main/kotlin/com/agoda/kakao/configurator/ConfiguratorBuilder.kt

```diff
@@ -0,0 +1,98 @@
+package com.agoda.kakao.configurator
+
+import android.support.test.espresso.DataInteraction
+import android.support.test.espresso.ViewInteraction
+import android.support.test.espresso.web.sugar.Web
+import com.agoda.kakao.delegates.DataInteractionDelegate
+import com.agoda.kakao.delegates.ViewInteractionDelegate
+import com.agoda.kakao.delegates.WebInteractionDelegate
+
+class ConfiguratorBuilder private constructor(
+    private val parentConfig: ConfigModel? = null
+) {
+
+    companion object {
+        fun clearCreate(): ConfiguratorBuilder = ConfiguratorBuilder()
```

Comment:
Naming convention. I think that just `create` will be enough

Reply:
I split into Configurator and Builder. In Builder in the constructor, you can set a history argument. So I removed all these companion functions, yep.